ballerina/api_inspectionplan_srv/Module.md

```diff
@@ -0,0 +1,80 @@
+
+The `ballerinax/sap.s4hana.api_inspectionlot_srv` package offers APIs for seamless integration with the [Inspection Lot API v1.2.0](https://api.sap.com/api/API_INFORECORD_PROCESS_SRV/overview).In the SAP system, you process quality inspections on the basis of inspection lots. The data relating to a quality inspection is grouped into several entities. With this service it is possible, for example, to update inspection lot header data or to create inspection results and usage decisions.
+
+## Setup guide
+
+1. Sign in to your S/4HANA dashboard.
+
+2. Under the `Communication Management` section, click on the `Display Communications Scenario` title.
+
+   ![Display Scenarios](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sap/main/docs/setup/3-1-display-scenarios.png)
+
+3. In the search bar, type `Quality Inspection Process Integration` and select the corresponding scenario from the results.
+
+   ![Search Sales Order](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sap/main/docs/setup/3-2-search-sales-order.png)
+
+4. In the top right corner of the screen, click on `Create Communication Arrangement`.
+
+   ![Click Create Arrangement](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sap/main/docs/setup/3-3-click-create-arrangement.png)
+
+5. Enter a unique name for the arrangement.
+
+   ![Give Arrangement Name](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sap/main/docs/setup/3-4-give-arrangement-name.png)
+
+6. Choose an existing `Communication System` from the dropdown menu and save your arrangement.
+
+   ![Select Existing Communication Arrangement](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sap/main/docs/setup/3-5-select-communication-system.png)
+
+7. The hostname (`<unique id>-api.s4hana.cloud.sap`) will be displayed in the top right corner of the screen.
+
+   ![View Hostname](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-sap/main/docs/setup/3-6-view-hostname.png)
+
+## Quickstart
+
+To use the `sap.s4hana.api_inspectionlot_srv` connector in your Ballerina application, modify the `.bal` file as follows:
+
+### Step 1: Import the module
+
+Import the `sap.s4hana.api_inspectionlot_srv` module.
+
+```ballerina
+import ballerinax/sap.s4hana.api_inspectionlot_srv as inspectionlot
+```
+
+### Step 2: Instantiate a new connector
+
+Use the hostname and credentials to initiate a client
+
+```ballerina
+configurable string hostname = ?;
+configurable string username = ?;
+configurable string password = ?;
+
+inspectionlot:Client inspectionlotClient = check new (
+    hostname = hostname,
+    config = {
+        auth: {
+            username,
+            password
+        }
+    }
+);
+```
+
+### Step 3: Invoke the connector operation
+
+Now, utilize the available connector operations.
+
+```ballerina
+inspectionlot:Wrapper createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();
```

Comment:
Inspection lot/ inspection plan? Review the docs.json. file fully

Reply:
Please find fix in [Update docs.json in two modules to fix automated document generation](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/10c211e4570ea2d2501780ca1029d936b69f092b)